### DISCUSSION TITLE
Anyone can help review PR in for the poetry buildpack

### DISCUSSION THREAD

**Discussion by dmitry-mightydevops** [paketo-buildpacks/feedback#285] [Python Team]
Hello,
https://github.com/paketo-buildpacks/poetry/pull/377

Can we ask for review of this PR to get it merged. Currently it's impossible to use poetry 1.8+ with paketo buildpacks. This PR fixes it.

**Comment by sophiewigmore** [comment]
Hey! Sorry for the delay, we've been a little bit light on maintainers recently. I am not a Python team member, but I have shared your PR with the right people

**Comment by dmitry-mightydevops** [comment]
thank you! @sophiewigmore
